preCICE-Ubuntu-20.04/install-basics.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -ex
+
+# We (may) need the multiverse repository for the VBox Guest Additions
+sudo apt-add-repository multiverse
+apt-get update
+sudo apt-get upgrade -qy
+
+# Install the Xfce desktop environment and basic applications
+sudo apt-get install -y xubuntu-core^
+sudo apt-get install -y thunar xfce4-terminal terminator bash-completion tree evince firefox firefox-locale-en
+
+# Install the VirtualBox guest additions
+sudo apt-get install -y virtualbox-guest-dkms virtualbox-guest-utils virtualbox-guest-x11
+
+# Use US-English keyboard layout
+L='us' && sudo sed -i 's/XKBLAYOUT=\"\w*"/XKBLAYOUT=\"'$L'\"/g' /etc/default/keyboard
```

Comment:
We need to provide a simple and interactive way of changing this. Preferably an icon directly on the desktop.

Reply:
The usual way would be that the user sets whatever they want (US, DE, Spanish, Italian, Greek, Klingon, ...) through the desktop environment tools. Most international keyboards follow the US layout, so I throught this is a good default. I think this belongs to an issue later.

Reply:
US as default is perfectly fine. More in an issue.

Reply:
I added a shortcut to the keyboard settings on the desktop.